fms/models/roberta.py

```diff
@@ -286,14 +286,19 @@ def factory(**kwargs):
 
 
 models.register_model(
-    _architecture_name, "micro", _roberta_factory_factory(_micro_char_config)
+    _architecture_name,
+    "micro",
+    _roberta_factory_factory(_micro_char_config),
+    _micro_char_config,
```

Comment:
do we actually need to register any of these configs or is this change only really applicable to mixtral?

Reply:
This is needed for Sphinx as well, we need to be able to get the type of model. If loading through safetensors, we don't have access to all of the weights, and cannot figure out which model variant we are adapting from

Reply:
my point was we don't actually need to change this for roberta/llama right?

Reply:
Should not need this for Roberta or Llama (for now), unless a new model comes along that has something like fused qkv that we need to support (could happen irregardless of the model architecture). In that case we would want to create an adapter for it that includes the model config.